lib/create-response-handler.ts

```diff
@@ -87,11 +95,16 @@ export interface ResponseScenarios {
 }
 
 /**
- * To make handling the passport.authenticate() callback easier we provide a createResponseHandler
- * function. This takes separate callbacks for each of the response scenarios you have to handle
- * and returns a function that will call the appropriate callback when called by passport.
+ * The `createResponseHandler` function should be used if your service uses the
```

Comment:
Should this 'should' be a 'must'?
Because if a VSP-only service uses `createResponseHandler`, it won't work, right?
(there might be other places where this applies, too)

Reply:
Yes, in the sense that you can only use this function and not the other function.  No, in the sense that you're not obliged to use either of them (although you're making more work for yourself if you choose not to).

Reply:
Fair, happy for it to stay as is.